op-signer/client/transaction_args.go

```diff
@@ -0,0 +1,85 @@
+// NOTE: most of the code below was pulled from go-ethereum/internal/ethapi/transaction_args.go
+// Main difference is it only expects DynamicFeeTxs
```

Comment:
QQ: What's the reason to pull in this source rather than using op-geth as a dependency? Is it mainly this `DynamicFeeTxs` thing?

Reply:
Ah, I see. I'll take a look at some of the other ways we do this.

Reply:
I took a look at this - I think pulling the tx args makes the most sense. It's annoying to have to do, but better than the alternatives.